components/org.wso2.carbon.identity.api.server.application.management/org.wso2.carbon.identity.api.server.application.management.v1/src/main/resources/applications.yaml

```diff
@@ -2445,6 +2445,27 @@ components:
           type: boolean
           description: Decides whether authorization policies needs to be engaged during authentication flows.
           example: true
+        additionalSpProperties:
+          $ref: '#/components/schemas/AdditionalProperties'
+    AdditionalProperties:
+      type: array
+      items:
+        $ref: "#/components/schemas/AdditionalSpProperty"
+    AdditionalSpProperty:
+        type: object
+        required:
+          - name
+          - value
+        properties:
+          name:
+            type: string
+            example: "isInternalApp"
+          value:
+              type: string
+              example: "true"
+          displayName:
+              type: string
+              example: "Internal Application"
```

Comment:
Do we need displayName. Do we have any requirement for this

Reply:
Not a mandatory field though